sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/data/SpanData.java

```diff
@@ -77,6 +78,15 @@
    */
   SpanId getParentSpanId();
 
+  /**
+   * Returns the parent {@code Context}. If the {@code Span} is a root {@code Span}, the Context
+   * will contain no Span or an invalid span.
+   *
+   * @return the parent {@code SpanId} or an invalid SpanId if this is a root {@code Span}.
+   * @since 0.8.0
+   */
+  Context getParent();
```

Comment:
Context isn't data, it's a way to move data around in-process, not out-of-process

Reply:
I disagree. The inflexibility of SpanContext/TraceState leaves no other choice to transmit nonstandard information from propagators to exporters. See https://github.com/open-telemetry/opentelemetry-java/pull/1611#discussion_r481846420

Reply:
I guess they are data, but not span data which may be what's confusing me. The context could have who knows what, maybe from the user, or some other library, and much of it wouldn't even be related to the span. I don't think it's a good idea to send everything in the Context to the exporter, Context is a bag for known values. If `SpanContext` had `List<Object>` that would make much more sense since it's know to be span data for sure.

Reply:
> If SpanContext had `List<Object>` that would make much more sense since it's know to be span data for sure.

I actually do like that idea. Or probably a `io.grpc.Context` on the SpanContext instead since it has a nice API for retrieving keys in a type-safe manner.

Reply:
> The context could have who knows what, maybe from the user, or some other library, and much of it wouldn't even be related to the span. I don't think it's a good idea to send everything in the Context to the exporter

I don't see a problem with that though. The exporter would just retrieve any values it knows (often only the parent span context) and ignore the rest.

Reply:
Context is no longer considered storeable data now.